core/src/main/java/com/novoda/espresso/TalkBackViewTestRule.java

```diff
@@ -1,15 +1,24 @@
 package com.novoda.espresso;
 
+import android.app.Instrumentation;
 import android.support.annotation.LayoutRes;
+import android.support.test.InstrumentationRegistry;
 import android.view.View;
 
 public class TalkBackViewTestRule<T extends View> extends ViewTestRule<T> {
 
-    private final TalkBackStateSettingRequester talkBackStateSettingRequester;
+    private final TalkBackStateSettingRequester talkBackStateSettingRequester = new TalkBackStateSettingRequester();
 
-    public TalkBackViewTestRule(@LayoutRes int id) {
-        super(id);
-        talkBackStateSettingRequester = new TalkBackStateSettingRequester();
+    public TalkBackViewTestRule(@LayoutRes int layoutId) {
+        this(new InflateFromXmlViewCreator<T>(layoutId));
```

Comment:
I had this one has a static ctor `withViewFromXml(@LayoutRes int layoutId)` initially, but inlined it into the ctor to avoid a breaking change.

Happy to pull it out to the static ctor or leave it here, since I can't think of another `ViewCreator` implementation that we'd be able to supply in the library to warrant named ctors.

Reply:
Is it possible to pull out the `talkback`? That way we can compose the tests with multiple rules instead of inheriting  (for the sake of the callback it seems?)

Reply:
Yes if we can guarantee that the TalkBack rule is called before the viewtestrule.

Reply:
we should be able to do it but it will require enforcing people to use `RuleChain` to order the rules.

will do it in another PR since it's pretty separate from this.

Reply:
https://github.com/novoda/espresso-support/issues/8

Reply:
Thanks :)